build-farm/make-adopt-build-farm.sh

```diff
@@ -31,6 +31,7 @@ if [ -z "$ARCHITECTURE"  ]; then
    if [ "$ARCHITECTURE" = "sparc"   ]; then ARCHITECTURE=sparcv9;    fi # Solaris/SPARC
    if [ "$ARCHITECTURE" = "powerpc" ]; then ARCHITECTURE=ppc64;      fi # AIX
    if [ "$ARCHITECTURE" = "arm"     ]; then ARCHITECTURE=aarch64;    fi # mac/aarch64
+   if [ "$ARCHITECTURE" = "loongarch64" ]; then ARCHITECTURE=loongarch64;    fi # loongarch64
```

Comment:
Can you also increase the whitespace indent for the `fi` parts of lines above and below so they line up again?